### ISSUE TITLE
Blackout start is set to the future by default

### ISSUE THREAD

**Issue by hasso** [alerta/alerta-webui#449]
**Issue Summary**
If user creates a blackout via UI, it starts in the future by default. Especially people who used an old UI for a long time don't expect it – they expect blackout start now. 

**Environment**
- OS: Linux
- API version: 8.3.3
- Deployment: self-hosted
- For self-hosted, WSGI environment: nginx/uwsgi

- Database: MongoDB

- Server config:
  Auth enabled? Yes
  Auth provider? Basic
  Customer views? No

- web UI version: 8.4.0

**To Reproduce**
1. Create a blackout at 11:47 for example.
2. If user doesn't change the start time, it is created with 12:00 as a start time.

**Expected behavior**
Users expect blackout start at the moment it is created most of the time.

**Comment by wo250012** [comment]
This has caused a few issues with our users as well. It would also be great to be able to specify a default blackout duration from the alerta.cfg file.

**Comment by satterly** [comment]
```
# blackout settings
BLACKOUT_DURATION = 3600  # default period = 1 hour
NOTIFICATION_BLACKOUT = False  # True - set alert status=blackout, False - do not process alert (default)
BLACKOUT_ACCEPT = []  # type: List[str]
# BLACKOUT_ACCEPT = ['normal', 'ok', 'cleared']  # list of severities accepted during blackout period
```
https://github.com/alerta/alerta/blob/master/alerta/settings.py#L250

**Comment by satterly** [comment]
@hasso @wo250012 both issues have been fixed in #471
